htdocs/supplier_proposal/class/supplier_proposal.class.php

```diff
@@ -781,15 +789,16 @@ public function updateline($rowid, $pu, $qty, $remise_percent, $txtva, $txlocalt
 			$this->line->fk_unit = $fk_unit;
 
 			// infos marge
-			if (!empty($fk_product) && $fk_product > 0 && empty($fk_fournprice) && empty($pa_ht)) {
+			// code seems useles fk_product is nerver defined and empty ....
+			/*if (!empty($fk_product) && $fk_product > 0 && empty($fk_fournprice) && empty($pa_ht)) {
```

Comment:
Should be better to replace fk_product by $line->fk_produce

Reply:
done